middleware/validate.js

```diff
@@ -0,0 +1,211 @@
+const knex = require('../db');
+knex('products')
```

Comment:
You should probably delete this line too since it looks like it was added as a sanity check more than functional code.